Formula/node.rb

```diff
@@ -105,7 +104,7 @@ def post_install
       prefix = #{HOMEBREW_PREFIX}
       python = /usr/bin/python\n
     EOS
-    (node_modules/"npm"/"npmrc").atomic_write npmrc
```

Comment:
Again with unrelated style changes …

Reply:
Well since I can no longer enforce my style onto the core via direct commit I've got to try and satisfy the habit somehow.